core/templates/pages/admin-page/services/admin-data.service.spec.ts

```diff
@@ -168,6 +168,46 @@ describe('Admin Data Service', () => {
     flushMicrotasks();
   }));
 
+  it('should send the math SVGs to the admin backend service', fakeAsync(() => {
+    var expectedResponse = {
+      result: 'success'
+    };
+    var latexToSvgMapping = {
+      exp_id1: {
+        latex_string1: {
+          file: new Blob(),
+          dimensions: {
+            height: '4d456',
```

Comment:
Not a valid height? (Ditto elsewhere)

Reply:
refer to this comment [link](https://github.com/oppia/oppia/pull/9985#discussion_r460495301)